Symfony/CS/Config/Config.php

```diff
@@ -53,6 +54,13 @@ public function setDir($dir)
         $this->dir = $dir;
     }
 
+    public function setCacheDir($cacheDir)
```

Comment:
This method needs to be added to the config interface. That makes it a breaking change, so this has to be done in 2.0.